### PR TITLE
enable upload file to consumer

### PR DESCRIPTION
**Pull request by huonggiangbui**
- Consumer can upload zip file with `file` in the body. The request id created by consumer will be referred as `match_id`
- The zip file will be stored under `event_id` container, with the name `match_<match_id>`
- Now to create a match record, consumer need to provide `match_id`, which is the request id created from above

### CODE REVIEW

File: server/Routes/Consumer/Consumer.py [hunk 1]
```diff
@@ -20,21 +20,30 @@ def check_auth(self):
...
             data['event_id'] = eventdata['_id']
+            blob_storage = BlobStorageModel()
+            container = blob_storage.get_container(str(eventdata['_id']))
+            if not container:
+                container = blob_storage.create_container(
+                    str(eventdata['_id']))
             with TeamsManager() as teamsmanager:
                 challenging_team = teamsmanager.find_team_by_id(ObjectId(data['team1_id']))
                 if challenging_team['last_challenge_timestamp'] == None:
                     with MatchRequestManager() as requestmanager:
```
Comment (KyrelJerome) [inline server/Routes/Consumer/Consumer.py:43]: The match result should be posted with a zip, seems this is posting for the request post?

Comment (SpaSmALX) [reply server/Routes/Consumer/Consumer.py:43]: Indeed, this post request shouldn't need to handle and zip files. Post the result zip to /api/match

Comment (huonggiangbui) [reply server/Routes/Consumer/Consumer.py:43]: I think consumer upload the zip so it should be done within post request right?

Comment (huonggiangbui) [reply server/Routes/Consumer/Consumer.py:43]: @KyrelJerome When a match is done (i.e. done the request), the id of the request will be the id of the match result (so you have to provide the match_id when create POST match). Then you can access the file with match_<match_id>

**Review by KyrelJerome — APPROVED** [review]

**Review by SpaSmALX — APPROVED** [review]

**Pull request merged**